plenum/server/node_bootstrap.py

```diff
@@ -99,11 +99,12 @@ def _init_common_managers(self):
                                                 ledger_sync_order=ledger_sync_order,
                                                 metrics=self.node.metrics)
 
-    def _register_pool_batch_handlers(self):
-        super()._register_pool_batch_handlers()
+    def _register_audit_batch_handlers(self):
         # TODO: This should be moved into LedgersBootstrap somehow
         future_primaries_handler = FuturePrimariesBatchHandler(self.db_manager, self.node)
-        self.write_manager.register_batch_handler(future_primaries_handler)
+        for lid in self.ledger_ids:
```

Comment:
I would make a comment that the order there is essential (that we need to register `FuturePrimariesBatchHandler` before the main AuditBatchHandler)